### PR TITLE
Restart ES to make AWS plugin work

### PR DESCRIPTION
**Pull request by JasonIamAUnixAdmin**

### CODE REVIEW

File: elasticsearch/aws_plugin.sls [hunk 1]
```diff
@@ -2,8 +2,9 @@ install_aws:
   cmd:
   - run
   - user: root
-  - name: '/usr/share/elasticsearch/bin/plugin install cloud-aws'
+  - name: '/usr/share/elasticsearch/bin/plugin install -b cloud-aws ; /sbin/service elasticsearch restart'
```
Comment (clarkperkins) [inline elasticsearch/aws_plugin.sls]: ## We shouldn't need the service restart, not?

After 3 different was of forcing a restart this is the one that worked. Other option is to do an ES service dead and then a start.

Comment (clarkperkins) [inline elasticsearch/aws_plugin.sls]: Also what does the `-b` do?

Comment (clarkperkins) [inline elasticsearch/aws_plugin.sls]: & what does the batch install flag do differently?

Comment (JasonIamAUnixAdmin) [inline elasticsearch/aws_plugin.sls]: the batch flag answers "yes" to the security question at the end of the install.

Comment (cpenner461) [inline elasticsearch/aws_plugin.sls]: If we do need the restart I think it'd be better to have a restart state using `service` and have it follow this state so it's more clear that installing this plugin will restart the service.

**Comment by JasonIamAUnixAdmin** [comment]
Change made and tested. Works as expected now.

**Comment by JasonIamAUnixAdmin** [comment]
This is good to go. Tested etc.

**Comment by cpenner461** [comment]
+1

**Pull request merged**